server/app/util/db.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose';
+import logger from './logger';
+
+export const connectDb = dbUrl => {
+  mongoose.connect(dbUrl);
+
+  // Mongo connection events
+  mongoose.connection.on('connected', () => {
+    logger.info(`💾  Mongo connected to ${dbUrl}`);
```

Comment:
Love the idea. The symbol is a bit confusing. If you have a symbol that better resembles a database then let's replace it. If not, we can keep it. :+1: